app/src/main/java/com/example/moviedetails/network/RetrofitClient.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.moviedetails.network
+
+import com.example.moviedetails.BASE_URL
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+object RetrofitClient {
+
+    fun getClient(): Retrofit {
```

Comment:
Looks like, it will create new client everytime we call `getClient`. Instead we can create one instance and use it.